src/app/_components/TherapyCard.js

```diff
@@ -2,8 +2,8 @@ import PropTypes from 'prop-types';
 import Image from 'next/image';
 import Link from 'next/link';
 import { PillButton } from '@components';
```

Comment:
🟢 change the alias to be @components/PillButton

Reply:
done, thanks